frontend/public/locales/en/translation.json

```diff
@@ -1357,9 +1357,8 @@
     "import.command.launchconsole": "Launch console",
     "import.command.pendingimport": "Cluster is pending import",
     "import.command.runcommand": "Run a command",
-    "import.credential.default": "Select credential mode",
-    "import.credential.explanation": "You need the kubeconfig file for the cluster that you're importing. Import information is used one time and is not saved.",
-    "import.description": "Once you click on \"Save import and generate code\", the information you entered is used to generate the code and cannot be modified anymore. If you wish to change any information, you will have to delete and re-import this cluster.",
+    "import.credential.explanation": "Credentials are stored in a secret temporarily until the cluster import begins.",
```

Comment:
Change to, "Credentials are temporarily stored in a secret until the cluster import begins."

Reply:
Will change.